src/Tasks/LockCheck.cs

```diff
@@ -211,7 +213,11 @@ internal static IEnumerable<ProcessInfo> GetLockingProcessInfos(params string[]
             const int maxRetries = 6;
 
             // See http://blogs.msdn.com/b/oldnewthing/archive/2012/02/17/10268840.aspx.
-            var key = new StringBuilder(new string('\0', CCH_RM_SESSION_KEY + 1));
+            char[] key = new char[CCH_RM_SESSION_KEY + 1];
+            for (int i = 0; i < key.Length; i++)
```

Comment:
This buffer will already be zeroed for you (unless the assembly has disabled zero unit globally)